include/evm_runtime/evm_contract.hpp

```diff
@@ -88,6 +88,8 @@ class [[eosio::contract]] evm_contract : public contract
    [[eosio::action]] void updtgasparam(eosio::asset ram_price_mb, uint64_t gas_price);
    [[eosio::action]] void setgasparam(uint64_t gas_txnewaccount, uint64_t gas_newaccount, uint64_t gas_txcreate, uint64_t gas_codedeposit, uint64_t gas_sset);
 
+   [[eosio::action]] void setgaslimit(std::optional<uint64_t> ingress_gas_limit);
+
```

Comment:
Why are we making the `ingress_gas_limit` optional?
I think we can just pass an `uint64_t`